### ISSUE TITLE
Unhandled error on invalid group type

### ISSUE THREAD

**Issue by PinKushin** [GandaG/fomod-validator#8]
Error information:

```
Version: 2.2.2
<class 'ValueError'>: '' is not a valid GroupType

  File "fomod_validator.py", line 259, in button_validate_clicked
  File "site-packages\pyfomod\parser.py", line 244, in parse
  File "site-packages\pyfomod\parser.py", line 205, in _iterparse
  File "site-packages\pyfomod\parser.py", line 112, in start
  File "c:\users\appveyor\.virtualenvs\fomod-validator-2qe23p5x\lib\enum.py", line 309, in __call__
  File "c:\users\appveyor\.virtualenvs\fomod-validator-2qe23p5x\lib\enum.py", line 561, in __new__
  File "c:\users\appveyor\.virtualenvs\fomod-validator-2qe23p5x\lib\enum.py", line 545, in __new__
  File "c:\users\appveyor\.virtualenvs\fomod-validator-2qe23p5x\lib\enum.py", line 574, in _missing_
```

I have 2 issues but this is the only one throwing an error code. The other FOMOD just won't run. Both are being created with FOMODCreationTool.

[ModuleConfig.txt](https://github.com/GandaG/fomod-validator/files/3668878/ModuleConfig.txt)
Is the xml, in plain text form, for the FOMOD throwing this error message.
Will try to rebuild the FOMOD with FOMODCreationTool to see if it's on my end, for some reason 2 step names keep going blank and I think that might have something to do with this.

[ModuleConfig.txt](https://github.com/GandaG/fomod-validator/files/3668891/ModuleConfig.txt)
Is the xml, in plain text form, for the FOMOD that will not run the Validator. I can separate these issues if you like, but I think they are related.

The one that wasn't loading into FOMOD Validator installed w/ Vortex w/o issue. I've reviewed, saved, and repackaged the file and the validator still will not run. 

The FOMOD throwing the error code would not install, 2 group names disappeared from the file somehow. That is probably what threw the error code. I think this may actually be an issue with FOMOD Creation Tool.

Yep, that was the issue. XML File had blank group areas. Validated properly after I fixed and saved it.
If I could, I would like to make the suggestion to handle this error in program. It would make debugging of the FOMOD package easier. Thank You.

The validator was running, I'm just an idiot who didn't realize when nothing is read out, it means there are no errors. A second suggestion if you would be so kind, make the program echo something to let the user know it ran and was successful. Thank you again, this tool really has been a lifesaver.

Edit: Inserted Code Markdown so the error would display correctly
Edit 2: Added More information.
Edit 3: Fix
Edit 4: I'm an idiot.

**Comment by GandaG** [comment]
Looks like that tool has a wee bug :p although could you try to install that with a mod manager? I'll still consider that invalid but it's nice to know what mod managers are accepting these days.

So the issue I noticed with the first file is that on line 155 the group type is "" which it can't be, the tool has to choose from a predefined list of options.

The second file also suffers from the above but what do you mean by won't run? You click the button and nothing happens?

Even though this does not seem related to this tool I'll leave it open to remind myself that I really need to improve the error reporting on validation.

**Comment by PinKushin** [comment]
[ModuleConfig.txt](https://github.com/GandaG/fomod-validator/files/3669302/ModuleConfig.txt)
[ModuleConfig.txt](https://github.com/GandaG/fomod-validator/files/3669309/ModuleConfig.txt)

Re-upload of the ModuleConfig.xmls for you. Both are now installing. The one that was throwing the error had, somehow, saved with blank options to check. I believe that was the issue throwing the Error. Once I deleted those it installed fine. Is the validator suppose to throw an all clear code if the FOMOD is clean? If not, I believe the Validator is working fine, though I would suggest printing out something on screen for a clean file so Users will know the application ran properly.

**Comment by dh-nunes** [comment]
Fixed in 1d5f906ac8b82a99948fe989a025dbaabd886c6c
